views/news/tabs.js

```diff
@@ -14,15 +15,11 @@ export default [
     },
   },
   {
-    id: 'PoliticOleNewsView',
-    title: 'PoliticOle',
-    rnVectorIcon: {iconName: 'megaphone'},
-    component: NewsContainer,
-    props: {
-      url: 'http://oleville.com/politicole/feed/',
-      mode: 'rss',
-      name: 'PoliticOle',
-    },
+    id: 'OlevilleView',
+    title: 'Oleville',
+    rnVectorIcon: {iconName: 'mouse-pointer'},
```

Comment:
`mouse-pointer` is not a valid icon for the tab bar, apologies. Tab bar uses ionicons, while the homescreen uses Entypo.